### PR TITLE
#163530774 API endpoint that registers users as candidates for election

### PR DESCRIPTION
**Pull request by femitj**
**What does this PR do?**

Creates api endpoint that registers users(politicians) as a candidate running for office

**Description of the task to be completed**

 - Users that has interest to run for office can register as a candidate

 - Ensure the following endpoint are working as they should
   - POST/api/v1/candidate
 
- Implement COMPOSITE PRIMARY key
   - using (user, office) to ensure a candidate is not registered twice
 
- secure route  

**How should this be tested manually**

 - After you clone the repo,
 - cd into it and run npm install to install node modules.
 - Make sure you have nodejs and express installed.
 - Afterwards, run npm start to initiate the app.
 - test endpoint above using postman with localhost:8080 as the host

**What are the relevant pivotal stories**
#163530774

### CODE REVIEW

File: src/controllers/Candidate.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      req.body.createdOn,
+    ];
+
+    try {
+      const { rows } = await db.query(createQuery, values);
+      return res.status(201).send({
+        status: 201,
+        data: [{
+          id: rows[0].id,
+          message: 'Candidate created',
```
Comment (andela-gike) [inline src/controllers/Candidate.js]: This according to the spec should return this 
image

File: src/routes/Routes.js [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+router.get('/api/v1/offices', Auth.verifyToken, OfficeController.getAllOffices);
+router.get('/api/v1/offices/:id', Auth.verifyToken, OfficeController.getOffice);
+
 // Users routes
 router.post('/api/v1/users', checkSignUpInput, userController.create);
 router.post('/api/v1/login', checkSignInInput, userController.login);
 
+// Candidate routes
+router.post('/api/v1/candidate', Auth.verifyToken, CandidateController.create);
+
```
Comment (andela-gike) [inline src/routes/Routes.js:31]: According to the spec this should be your route `POST /office/<user-id>/register`

**Review by andela-gike — CHANGES_REQUESTED** [review]

File: src/controllers/Candidate.js (hunk 1 above)
Comment (femitj) [reply src/controllers/Candidate.js]: done

File: src/routes/Routes.js (hunk 2 above)
Comment (femitj) [reply src/routes/Routes.js:31]: done

**Review by andela-gike — APPROVED** [review]

**Pull request merged**